ivy_tests/test_ivy/test_frontends/test_paddle/test_fft.py

```diff
@@ -105,3 +105,39 @@ def test_paddle_ifft(
         axis=axis,
         norm=norm,
     )
+@handle_frontend_test(
+    fn_tree="paddle.fft.fft2",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_value=-10,
+        max_value=10,
+        min_num_dims=1,
+        min_dim_size=2,
+        valid_axis=True,
+        force_int_axis=True,
+    ),
+    n=st.one_of(
+        st.integers(min_value=2, max_value=10),
+        st.just(None),
+    ),
+    norm=st.sampled_from(["backward", "ortho", "forward"]),
+)
+def test_paddle_fft2(
+    dtype_x_axis,
+    n,
+    norm,
+    frontend,
+    test_flags,
+    fn_tree,
+):
+    input_dtypes, x, axes = dtype_x_axis
```

Comment:
`axes` is incorrectly specified in the tests, axes should be in range of `[-num_dims, num_dims` . For a tensor of shape (3,3,3) , `num_dims` is `3` . hence the range of axes is `[-3,3]`. Also make sure to have the correct type for this argument. See [here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/fft/fft2_en.html)  for the types and more.